### PR TITLE
Hughsie/dashdash fast fail

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util.c [hunk 1]
```diff
@@ -4869,6 +4903,14 @@ main(int argc, char *argv[])
...
 	     N_("Answer yes to all questions"),
 	     NULL},
+	    {"fast-fail",
+	     'y',
+	     0,
+	     G_OPTION_ARG_NONE,
+	     &priv->fast_fail,
+	     /* TRANSLATORS: command line option */
+	     N_("Return status as soon as one update has failed"),
+	     NULL},
```
Comment (superm1) [inline src/fu-util.c:4913]: I'd argue we want the inverse.  When do you ever want to let it keep going it if fails?  Debugging only I'd think.

Comment (hughsie) [reply src/fu-util.c:4913]: Ohh I use it for "how many emulations are broken by this change" but you're completely correct. Should I add a [secret?] command line argument (`--keep-going?`) or just rip out the "keep going" functionality completely?

Comment (superm1) [reply src/fu-util.c:4913]: How about a secret option only for unsupported build?

**Review by superm1 — DISMISSED** [review]

**Comment by superm1** [comment]
> @superm1 any idea why debian i386 got `ModuleNotFoundError: No module named 'dbusmock'`?

Huh?  It seems it passed?

**Review by superm1 — APPROVED** [review]

**Comment by hughsie** [comment]
> Huh? It seems it passed?

It's a *ghost*. Sorry for the noise.

**Pull request merged**